apps/frontend/src/utils/helperFunctions.tsx

```diff
@@ -231,3 +231,9 @@ export function fromArrayToCommaSeparated(
 ) {
   return itemsArray?.map((item) => item ?? '-').join(', ') || '-';
 }
+
+export const isLegacyCall = (cycleComment: Scalars['String']['input']) => {
+  return cycleComment === 'Created during legacy proposal import'
```

Comment:
This is looking more STFC specific. When doing globally/generically, I am not sure if using the Cycle Comment is the right approach to decide something. 

Just a thought: Is it possible to handle this in backend through STFC resolvers and inject it, so that it doesn't affect the main codebase.